ci/run-tests.ps1

```diff
@@ -3,7 +3,9 @@ param(
     [string] $uproject_path,
     [string] $test_repo_path,
     [string] $log_file_path,
-    [string] $test_repo_map
+    [string] $test_repo_map,
+    [string] $tests_path = "SpatialGDK",
+    [bool] $override_spatial_networking = $true
```

Comment:
What if we want to use the project's default setting for spatial networking?
With this flow we will always override it either to true or false.

Reply:
@mironec  do we have any tests like this already? I assumed that we always run tests with spatial networking and as a special case we can run native unreal tests.

Reply:
It's true that we do not and probably do not want to, it's probably okay to leave it in.
If we ever have a project that is default native that we want to run tests against in CI, we can revisit this.